src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,39 @@
+package step2.domain;
+
+import java.util.List;
+
+/**
+ * 로또 엔터티
+ */
+public class Lotto {
+    private final static int LOTTO_PRICE = 1000;
+    private final LottoNumbers lottoNumbers;
+
+    public Lotto() {
+        this.lottoNumbers = new LottoNumbers();
```

Comment:
```suggestion
        this.lottoNumbers = LottoNumbers.auto();
```

이 코드를 봤을 때, 내부 구현을 보고 이해할 수 있는 코드와
흐름만 봐도 쉽게 파악할 수 있는 코드 중 어떤 쪽을 선호하시나요?

개발한 본인 외에 팀원들이 봤을 때 더 직관적으로 이해하기 쉬운 코드는 어떤 모습일지 고민해보면 좋을 것 같아요.
(`LottoNumbers.auto()`는 예시일 뿐입니다~ )